native/cocos/renderer/pipeline/custom/FGDispatcherTypes.h

```diff
@@ -169,7 +168,7 @@ struct ResourceAccessGraph {
         return {_vertices.get_allocator().resource()};
     }
 
-    inline boost::container::pmr::memory_resource* resource() const noexcept {

```

Comment:
Why remove inline?

Reply:
`inline` is unnecessary in class member function declaration. `clangd` will report warning.

Reply:
Ok, may be we should remove all inline in future.